homeassistant/helpers/trigger.py

```diff
@@ -68,6 +69,48 @@ class TriggerInfo(TypedDict):
     trigger_data: TriggerData
 
 
+class PluggableAction:
+    """A pluggable action handler."""
+
+    def __init__(self, update: CALLBACK_TYPE | None = None) -> None:
```

Comment:
I have made `update` optional, to handle both cases (`philips_js` with, and `webostv` without)